client/src/App.js

```diff
@@ -1,66 +1,20 @@
 import React, { Component } from "react";
-import PropTypes from 'prop-types';
+// import Nav from "./components/Nav/Nav";
 // import { Navbar, Button } from "react-bootstrap";
 // import Auth from "./components/Authorization/Authorization";
 import Main from "./pages/Main";
 import Profile from "./components/Profile/Profile";
-import Social from "./pages/Social";
 import "./App.css";
+import PropTypes from "prop-types";
 import { withStyles } from "@material-ui/core/styles";
 import AppBar from "@material-ui/core/AppBar";
 import Toolbar from "@material-ui/core/Toolbar";
 import Typography from "@material-ui/core/Typography";
 import Button from "@material-ui/core/Button";
 import IconButton from "@material-ui/core/IconButton";
 import MenuIcon from "@material-ui/icons/Menu";
-import Menu from '@material-ui/core/Menu';
-import MenuItem from '@material-ui/core/MenuItem';
-import ListItemIcon from '@material-ui/core/ListItemIcon';
-import ListItemText from '@material-ui/core/ListItemText';
```

Comment:
This is for the menu items in the hamburger drop down. We will need this if we are keeping the hamburger.